spec/changelog.yml

```diff
@@ -7,6 +7,9 @@
   - description: Add support for subobjects in fields
     type: enhancement
     link: https://github.com/elastic/package-spec/pull/573
+  - description: Add support for counted_keyword field type in data streams
+    type: enhancement
+    link: https://github.com/elastic/package-spec/pull/707
```

Comment:
I just noticed that the `Add validation for ingest pipeline rename processor` entry in `3.0.5-next` has a double `type` key: I guess this will likely break the release, shall I fix it in this PR?

I assume the `type` for it should be `breaking-change` instead of `enhancement`?

Reply:
Yep, this looks incorrect, feel free to make the change here, if not we will do it when we prepare the changelog on next version.